md/reshade_installation.ja.md

```
@@ -9,26 +9,129 @@
+> 「詳細情報」をクリックし、「実行」ボタンを選択してください。
+
+警告ダイアログが表示される場合は「OK」を押下します：
+
+> **ReShade 警告メッセージ:**
+> ```
+> ReShade is intended for single-player games only. Use in online games may be considered cheating and could result in permanent bans. Use at your own risk.
+> ```
+> 
+> この警告は「シングルプレイゲームでの利用が意図されており、マルチプレイゲームで利用してBANされても自己責任です」という内容です。
+
 <img src='https://github.com/gposingway/gposingway/assets/18711130/6a57b0d1-5684-441b-94b3-01254d38095a' width='408' /><br/><br/>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)